common/components/chrome/MainHeader.jsx

```diff
@@ -85,307 +52,139 @@ class MainHeader extends React.Component<{||}, State > {
-                        My Projects
-                      </div>
-                    </a>,
-                    <Divider />
-                    ]
-                  }
-                </div>
-
-              }
-              <a href={url.section(Section.FindProjects, {hideSplash: 1})}>
-                <div className={'SubHeader-drawerDiv'} >
-                  Find Projects
-                </div>
-              </a>
-              <Divider />
-
-              <a href={this.state.createProjectUrl}>
-                <div className={'SubHeader-drawerDiv'} >
-                  Create Project
-                </div>
-              </a>
-              <Divider />
-
-              <a href={url.section(Section.AboutUs)}>
-                <div className={'SubHeader-drawerDiv'} >
-                  About Us
-                </div>
-              </a>
-              <Divider />
-
-              <a href={url.section(Section.Press)}>
-                <div className={'SubHeader-drawerDiv'} >
-                  News
-                </div>
-              </a>
-              <Divider />
-
-              {this._renderHamburgerNavigationLinks()}
-
-              <Divider />
-              {
-                !CurrentUser.isLoggedIn() &&
-                <React.Fragment>
-                  <div className={'SubHeader-drawerDiv'} onClick={this._onLogInClick}>
-                    <a href="" > Log In </a>
-                  </div>
-                  <Divider />
-                </React.Fragment>
-              }
-              {
-                CurrentUser.isLoggedIn() &&
-                <a href="/logout/">
-                  <div className={'SubHeader-drawerDiv'}>
-                    Logout
-                  </div>
-                </a>
-              }
-            </div>
-          </div>
-        </Drawer>
+        <Button className="MainHeader-showdesktop MainHeader-donate-link" variant="link" href={url.section(Section.Donate)}>Donate</Button>
+        <Button className="MainHeader-showdesktop MainHeader-login-button" variant="outline-primary" href={url.section(Section.LogIn, url.getPreviousPageArg())}>Log In</Button>
       </React.Fragment>
-
     )
   }
 
-  _renderHamburgerNavigationLinks(): $ReadOnlyArray<React$Node> {
-    //TODO: refactor NavigationLinks to define link behavior on a per-link basis
-    return NavigationLinks.list().map((link) => {
-      return (
-        <React.Fragment key={link.url}>
-          <a key={link.url} href={link.url} rel="noopener noreferrer" onClick={NavigationLinks.logClick.bind(this, link)}>
-            <div className={'SubHeader-drawerDiv'}>
-              {link.name}
-            </div>
-          </a>
-          <Divider />
-        </React.Fragment>
-      );
-    });
+//TODO: Refactor these to reduce duplication
+//TODO: Allow multiple arguments for url.section to handle url.getPreviousPageArg() - options object?
+  _renderNavLink(sec, text, classes = "") {
```

Comment:
nit: "sec" isn't a universally understood shorthand for 'section', and it only saves 4 characters, so I'd just use 'section' here